modules/profile/manifests/mysqlserver/replication.pp

```diff
@@ -26,8 +26,13 @@
   bind_address        => $bind_address,
 }
 
+$repo_descr = $mysql::install::repo_descr
```

Comment:
Variables shouldn't go from module.
All data should be defined on PROFILE level.